providers/apis/defi/uniswapv3/utils.go

```diff
@@ -96,6 +97,44 @@ var (
 		URL:              "https://eth.public-rpc.com/",
 	}
 
+	// DefaultBaseAPIConfig is the default configuration for the Uniswap API. Specifically this is for
+	// Base mainnet.
+	DefaultBaseAPIConfig = config.APIConfig{
+		Name:             fmt.Sprintf("%s%s%s", BaseName, NameSeparator, constants.BASE),
+		Atomic:           true,
+		Enabled:          true,
+		Timeout:          1000 * time.Millisecond,
+		Interval:         2000 * time.Millisecond,
+		ReconnectTimeout: 2000 * time.Millisecond,
+		MaxQueries:       1,
+		URL:              "https://mainnet.base.org",
+	}
+
+	// DefaultBaseMarketConfig is the default market configuration for Uniswap V3. Specifically
+	// this is for Base mainnet.
+	DefaultBaseMarketConfig = types.CurrencyPairsToProviderTickers{
```

Comment:
Can we remove the default market config? My plan is to deprecate all of these + use our static marketmaps for testing